src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.ApplicationModel.Configuration/ConfigurationClientModelFactory.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Azure.ApplicationModel.Configuration
+{
+    public static class ConfigurationClientModelFactory
+    {
+        public static ConfigurationSetting ConfigurationSetting(
```

Comment:
Should these be called Create<return_type>?

Reply:
I followed Roslyns `SyntaxFactory` (https://docs.microsoft.com/en-us/dotnet/api/microsoft.codeanalysis.csharp.syntaxfactory?view=roslyn-dotnet) pattern but have no strong opinion.

Reply:
The type name was slightly long so I went with shorter method names.